tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -1201,19 +1217,36 @@ class CudnnRnnSequenceTensorDescriptor
                                    cudnnDataType_t data_type,
                                    TensorDescriptor handle)
       : parent_(parent),
-        seq_length_(seq_length),
+        max_seq_length_(seq_length),
         batch_size_(batch_size),
         data_size_(data_size),
         data_type_(data_type),
         handle_(std::move(handle)),
         handles_(seq_length, handle_.get()) {}
 
+  CudnnRnnSequenceTensorDescriptor(CUDAExecutor* parent, int seq_length,
```

Comment:
It seems unclear who owns `seq_lens`, the caller or callee?

How about using a `absl::Span` instance? Also, I prefer spelling out full name `seq_lengths`